src/TemplateElement.js

```diff
@@ -1,12 +1,12 @@
-import { StyledElement } from './StyledElement';
-import { html } from 'lit-html';
-import { render } from 'lit-html/lib/shady-render.js';
-export { i18n } from './util/i18n';
+import { StyledElement } from './StyledElement.js';
+import { html, attr, unsafeHTML } from './util/html.js';
+import { render } from './util/render.js';
+export { i18n } from './util/i18n.js';
 
 class TemplateElement extends StyledElement {
 	constructor(options) {
 		super({
-			deferUpdate: false,
+			deferUpdate: true,
```

Comment:
I think this will/would change the default behavior when using the TemplateElement without SSR. I think we should use the attribute "defer-update" instead. That can be set easily during SSR and won't change anything when using the TemplateElement client side only.

The behaviour is already implemented in the BaseElement:
```javascript
if (this.hasAttribute('defer-update') || this._options.deferUpdate) {
	// don't updates/render, but register refs and events
	this.registerEventsAndRefs();

	this.triggerHook('connected');
} else {
	this.requestUpdate({ notify: false }).then(() => {
		this.triggerHook('connected');
	});
}
```

Reply:
@eddyloewen If we are setting the attribute, the developer has no chance to override the behaviour, if desired. But i agree that the default of `false` should not be changed.

Reply:
@lukas-schardt Yeah that's right. Maybe we can make the SSR attribute configurable? Or maybe instead of checking for the "defer-update" attribute we could check if the element already has content/children? But that might have other implications.

Reply:
@eddyloewen I thought about it. what if we set the defer-update attribute in the ssr renderer, only if it hasn't been overridden in the constructur explicitly? This way we would respect the users choice, but the ssr framework could set the default and the client does not have to made aware that it could have been prerendered on the server